solr/solr-ref-guide/modules/query-guide/pages/faceting.adoc

```diff
@@ -113,7 +113,7 @@ This does not limit the query in any way, only the facets that would be returned
 |Optional |Default: none
 |===
 +
-If `facet.contains` is used, the `facet.contains.ignoreCase` parameter causes case to be ignored when matching the given substring against candidate facet terms.
+Causes case to be ignored when matching the `facet.contains` substring against candidate facet terms.
```

Comment:
nit/subjective
```suggestion
If set to `true`, causes case to be ignored when matching the `facet.contains` substring against candidate facet terms.
```

Reply:
Nice, this appears to be consistent with other lines on this page.